### PR TITLE
modified long/short arrows in equation mode

### PR DESCRIPTION
**Pull request by faenuccio**
adapted all short arrows $\to$ and $\mapsto$ to long ones  $\longrightarrow$ and $\longmapsto$ when in equation mode

### CODE REVIEW

File: src/condensed.tex [hunk 1]
```diff
@@ -330,17 +330,17 @@ \subsection{Condensed abelian groups}
   \leanok
   By Proposition~\ref{CHPNG-Cond-exact} it suffices to show
   that the corresponding sequence of pseudonormed groups is short exact,
-  and by Proposition~\ref{exact-with-constants-lim} we may also assume that $S$ is finite.
+  and by Proposition~\ref{exact-with-constants-lim} we may also assume that $S$ is finite. With these reductions, the result is Theorem~\ref{theta-exact_finite}.
```
Comment (jcommelin) [inline src/condensed.tex]: The PR seems to be doing a bit more than it promised :wink:

File: src/condensed.tex [hunk 2]
```diff
@@ -330,17 +330,17 @@ \subsection{Condensed abelian groups}
...
-  For finite $S$, the sequence is a termwise direct sum (indexed by $s \in S$) of the sequences
-  \[ 0 \to \Lm_{r'}(\{s\}) \to \Lm_{r'}(\{*\}) \to \Rm(\{*\}) \to 0. \]
-  Hence it suffices to show that the sequence
-  \[ 0 \to \Z(\!(T)\!)_{r'} \to \Z(\!(T)\!)_{r'} \to \R \to 0 \]
-  is short exact.
-  The Lemmas~\ref{theta-surjective} and~\ref{theta-continuous} show that the evaluation map is indeed
-  continuous and surjective.
-  A computation (omitted here for now, but done in Lean) shows that the kernel of the evaluation map
-  is exactly $(2T-1) \Z(\!(T)\!)_{r'}$.
+%  For finite $S$, the sequence is a termwise direct sum (indexed by $s \in S$) of the sequences
```
Comment (jcommelin) [inline src/condensed.tex]: Please revert this for now.

File: src/condensed.tex [hunk 3]
```diff
@@ -264,20 +264,20 @@ \subsection{Condensed abelian groups}
   \lean{Lbar.short_exact}
   \leanok
   Let $0 < r < 1$ be a real number, and let $S$ be a profinite set.
-  Decomposing $\mathbb Z((T))_r$ into positive and nonpositive coefficients yields a direct sum decomposition
-  \[ \mathbb Z((T))_r = T\mathbb Z[[T]]_r \oplus \mathbb Z[T^{-1}]. \]
+  Decomposing $\mathbb Z(\!(T)\!)_r$ into positive and nonpositive coefficients yields a direct sum decomposition
```
Comment (jcommelin) [inline src/condensed.tex]: These changes are fine. But maybe add them to the PR-title and/or top post.

File: src/macros.tex [hunk 4]
```diff
@@ -90,10 +90,10 @@
 \newcommand{\qproet}{{\mathrm{qpro\acute{e}t}}}
 \newcommand{\BC}{{\mathcal B\mathcal C}}
 \newcommand{\heart}{\heartsuit}
-\def\F{\mathbf{F}}
-\def\N{\mathbf{N}}
-\def\Z{\mathbf{Z}}
-\def\R{\mathbf{R}}
+\def\F{\mathbb{F}}
```
Comment (jcommelin) [inline src/macros.tex]: Idem dito.

File: src/measures.tex [hunk 5]
```diff
@@ -5,36 +5,34 @@ \subsection{Spaces of Measures}
   \lean{real_measures}
   \leanok
   Let $0 < p' < 1$ be a real number, and let $S$ be a finite set.
-  Then $\Rm(S)$ denotes the real vector space
```
Comment (jcommelin) [inline src/measures.tex]: Please revert these changes for now.

File: src/measures.tex [hunk 6]
```diff
@@ -49,21 +47,22 @@ \subsection{Spaces of Measures}
 	\Lm_{r'}(S)_{A,\leq c}=\bigl\{\sum_{s\in S,n\in A}a_{n,s}T^n[s] \mid a_{n,s}\in \Z, \sum_{n\in A, s\in S}\lvert a_{n,s}\rvert r^n\leq c\bigr\}.
 \]
 This defines a profinite topology on $\Lm_{r'}(S)_{\leq c}$, for each $c>0$.
-
-\begin{definition}
```
Comment (jcommelin) [inline src/measures.tex]: Please revert for now.

File: src/measures.tex [hunk 7]
```diff
@@ -72,6 +71,10 @@ \subsection{Spaces of Measures}
 %  \alpha\geq 0 \Longrightarrow \lfloor \alpha\rfloor \geq 0.
 %\]
 
+%Now fix $0< \xi<1$ and let $x\in\R_{\geq 0}$.
```
Comment (jcommelin) [inline src/measures.tex]: Idem

File: src/measures.tex [hunk 8]
```diff
@@ -192,35 +195,29 @@ \subsection{Spaces of Measures}
 %  The result follows from Lemma~\ref{theta.limit_y}.
 %\end{proof}
 
-Now fix $0< \xi<1$ and let $x\in\R_{\geq 0}$. For simplicity, denote by $\Z(\!(T)\!)_{r'}$ the group $\Lm_{r'}(\ast)$.
-One of the key results proven in \cite[\S 6]{Analytic} is the surjectivity and continuity of the following map.
+%Now fix $0< \xi<1$ and let $x\in\R_{\geq 0}$. For simplicity, denote by $\Z(\!(T)\!)_{r'}$ the group $\Lm_{r'}(\ast)$.
```
Comment (jcommelin) [inline src/measures.tex]: Idem

File: src/measures.tex [hunk 9]
```diff
@@ -192,35 +195,29 @@ \subsection{Spaces of Measures}
...
+%One of the key results proven in \cite[\S 6]{Analytic} is the surjectivity and continuity of the following map.
 
 \begin{definition}
   \label{theta}
   \lean{theta.ϑ}
   \uses{laurent-measures}
   \leanok
-  Fix $0<\xi <1$, and let $\vartheta_{\xi}\colon \Z(\!(T)\!)_r\rightarrow \R$ be the evaluation map
+  Let $0<\xi <1$ be a real number. Denote by $\vartheta_{\xi}\colon \Z(\!(T)\!)_{r'}\rightarrow \R$ the evaluation map
```
Comment (jcommelin) [inline src/measures.tex]: Idem

File: src/measures.tex [hunk 10]
```diff
@@ -192,35 +195,29 @@ \subsection{Spaces of Measures}
...
   \[
     \sum a_nT^n\longmapsto \sum a_n\xi^n.
   \]
+induced by $T\mapsto \xi$.
 \end{definition}
 
-In order to prove the surjectivity, we use the following construction.
```
Comment (jcommelin) [inline src/measures.tex]: Idem

**Review by jcommelin — COMMENTED** [review]
This PR is making a lot of changes beyond long/short arrows. Could you please take another look?

**Comment by faenuccio** [comment]
OK, I think I have reverted all changes unrelated to arrows. I hope I got it right this time, sorry for the noise!

**Review by jcommelin — APPROVED** [review]
Thanks! Looks good to me.

But I'm a bit weary to merge this if the blueprint can't be rebuild due to that `list_decls` bug. So I'll wait a bit with merging.

**Comment by faenuccio** [comment]
Sure! I will move on (tomorrow, probably) to all modifications concerning $\bigcup$ and $\varinjlim$ at any rate.
